packages/http/src/mocker/negotiator/NegotiatorHelpers.ts

```diff
@@ -135,6 +134,15 @@ const helpers = {
     const { mediaTypes, dynamic, exampleKey } = desiredOptions;
 
     return withLogger(logger => {
+      if (_httpOperation.method === 'head') {
+        logger.warn(`Responding with an empty body to a HEAD request.`);
```

Comment:
This should not be a warn, there's nothing to be worried about. We can use `info`